src/app/redirects.test.js

```
@@ -26,12 +26,12 @@ const expectWithNewRoute = async (route, expectation, cb) => {
   try {
     await resolveRedirects()
     cb(() => expect(window.location.replace))
-  } catch (e) {
-    // console.log(e) hide unexpected async errors from Jest
+  } catch {
+    // no-op
```

This is a bit opaque. Maybe define why catch shouldn't do anything?

---

This was existing code. I didn't look into why a `try/catch` block is necessary here.

---

I think the empty catch is to `hide unexpected async errors from Jest`, right? That's a bit more descriptive than `no-op`

---

It's pretty clear that an empty `catch` does that; the comment doesn't add anything we don't already know.

---

I disagree, I think the original comment makes it obvious that we're intentionally not showing something. `no-op` just means nothing is happening there.